packages/gtk3.rb

```diff
@@ -35,8 +22,11 @@ class Gtk3 < Package
   depends_on 'shared_mime_info'
   depends_on 'six' => :build
   depends_on 'xdg_base'
+  depends_on 'llvm'
```

Comment:
`depends_on 'llvm' => :build`?  Or is this also a runtime dependency?

Reply:
Ah yes, that's what I meant.